messages/ru/yii-validator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'The model is not valid. Must have at least "{min}" filled attributes.' => 'Модель должна содержать минимум {min, number} {min, plural, one{заполненный атрибут} few{заполненных атрибута} many{заполненных атрибутов} other{заполненных атрибута}}.',
```

Comment:
"The model" is a bit confusing because arrays are also acceptable. So maybe replace it with something more abstract or omit completely? Original message needs to be also corrected.

Reply:
Goal of this PR — translation only. Suggest make this later.

Reply:
#414